Source/Initialization/PlasmaInjector.H

```diff
@@ -73,6 +73,7 @@ public:
 
     bool external_file = false;
     std::string str_injection_file;
+    amrex::Real physical_q_tot = 0.0;
```

Comment:
Shall we add a member-variable description, because there are so many?
```suggestion
    amrex::Real physical_q_tot = 0.0; //! re-scale an externally loaded particle beam to this total charge
```

Reply:
How about this? (I am scared of calling what I am getting a beam ... sorry, it's the physicist in me that complains!)
```suggestion
    amrex::Real physical_q_tot = 0.0;  //! re-scale weight of externally loaded particles to this total charge
```